mtbf_driver/mtbf_apps/settings/app.py

```diff
@@ -21,7 +22,7 @@ def wait_for_cellanddata(self):
         self.wait_for_condition(lambda m: m.find_element(*self._cellanddata_menu_locator).get_attribute('aria-disabled') != 'true')
 
     def go_back(self):
-        self.wait_for_element_displayed(*self._header_locator)
+        Wait(self.marionette, timeout=120).until(expected.element_displayed(*self._header_locator))
         header = self.marionette.find_element(*self._header_locator)
         # TODO: replace this hard coded value with tap on the back button, after Bug 1061698 is fixed
```

Comment:
ref to the current gaiatest, we might still need to use the location to tap..so I'll leave this here without any changes